src/server/test_utils.cc

```diff
@@ -404,6 +404,11 @@ RespExpr BaseFamilyTest::Run(ArgSlice list) {
   return Run(GetId(), list);
 }
 
+RespExpr BaseFamilyTest::Run(std::string_view command) {
```

Comment:
Was there a reason we didn't support that ? It's so much nicer to just cal `"command arg1 arg2 arg3"`